happypose_ros/test/test_initialization_params.py

```diff
@@ -103,6 +106,7 @@ def test_leading_publish_tf(happypose_params: dict) -> None:
             **happypose_params,
             parameter_overrides=[
                 Parameter("cosypose.dataset_name", Parameter.Type.STRING, "ycbv"),
+                Parameter("cosypose.model_type", Parameter.Type.STRING, "synth+real"),
```

Comment:
This test is not required in this case, as we assume the default behaviour is `pbr`. This test only checks fields that are absolutely obligatory for the end user to provide

Reply:
ok, will remove this line then